node/hello-world/index.js

```diff
@@ -0,0 +1,65 @@
+// A version of this script is available for node v4 in index.v4.js
+
+const bigtable = require('@google-cloud/bigtable');
+
+const TABLE_NAME = 'Hello-Bigtable';
+const COLUMN_FAMILY_NAME = 'cf1';
+const COLUMN_NAME = 'greeting';
+const INSTANCE_ID = process.env.INSTANCE_ID;
+
+if (!INSTANCE_ID) {
+  console.error(
+    'Environment variables for INSTANCE_ID must be set!'
+  );
+  process.exit(1);
+}
+
+const getRowGreeting = (row) => {
+  return row.data[COLUMN_FAMILY_NAME][COLUMN_NAME][0].value;
+};
+
+(async () => {
+  try {
+    const bigtableClient = bigtable();
+    const instance = bigtableClient.instance(INSTANCE_ID);
+
+    /* If the table already exists, add // to the start of this line.
+    const table = instance.table(TABLE_NAME);
+    /*/
+
+    console.log(`Creating table ${TABLE_NAME}`);
+    const options = {
+      families: [COLUMN_FAMILY_NAME],
+    };
+    const [table] = await instance.createTable(TABLE_NAME, options);
```

Comment:
Is there a way if the table already exists?  There's a GetTable RPC which can be used for that functionality, but I'm not sure if it's exposed through our API.  If it's not, that's an API bug.

Reply:
The only way the table can exists if the script errored out from someone tinkering with it. If that happens, I wanted to make sure there was a simple solution to recover without having to use cbt or use some other mechanism. I figured this was a quick start example and didn't want someone to get bogged down with something like this

Reply:
Great idea! I added an exists check and separate `create()` call